src/integrators/integrator_utils.jl

```diff
@@ -161,6 +161,22 @@ function stepsize_controller!(integrator,alg)
   end
   q
 end
+
+function stepsize_controller!(integrator,alg::OrdinaryDiffEqAdamsVarOrderVarStepAlgorithm)
```

Comment:
This isn't necessary. It's exactly the same as the other one which would be called if this doesn't exist. Is there a reason you added it?

Reply:
`beta1` and `beta2` are calculated once in starting after that order is changing so `beta1` and `beta2` should be updated.

Reply:
I'm not sure that's actually stable though. The PI gains are generally assumed constant. Instead, just add this algorithm to https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/integrators/integrator_utils.jl#L172 for now so that way it doesn't have the PI part. We can play with the extended controllers later.

Reply:
means we should use `std` stepsize controller for now?

Reply:
yes for now.

Reply:
eliminate what's not necessary to get it working, and then tag on new stuff over time to improve it. Starting with the whole setup, especially when that algorithm together has never been tested or analyze could cause debugging headaches.

Reply:
Okay